.eslintrc

```diff
@@ -7,6 +7,11 @@
     "es6": true,
     "node": true
   },
+  "globals": {
```

Comment:
Please add /* global $, sessionStorage */ to userModel.js and import Backbone.
Using globals in eslint config is bad.